shoes-core/lib/shoes/common/style.rb

```diff
@@ -139,6 +140,11 @@ def set_visibility(new_styles)
         update_visibility
       end
 
+      def set_coloring(new_styles)
+        update_fill   if new_styles.include?(:fill)
+        update_stroke if new_styles.include?(:stroke)
+      end
```

Comment:
Agreed on the common styling making this so much easier! If we get a few more I'm imagining a callback system of some sort, but it will be easy to plumb in thanks to @KCErb's hard work.